app/controllers/Reviews.scala

```diff
@@ -76,6 +76,27 @@ final class Reviews @Inject()(data: DataHelper,
     }
   }
 
+  def reopenReview(author: String, slug: String, versionString: String) = {
+    (Authenticated andThen PermissionAction[AuthRequest](ReviewProjects)) async { implicit request =>
+      withProjectAsync(author, slug) { implicit project =>
+        withVersionAsync(versionString) { version =>
+          version.mostRecentReviews.map(_.headOption).map {
+            case None => NotFound
+            case Some(review) =>
+              this.users.get(-1).map { systemUser =>
```

Comment:
Why get the system user here when it's not used?

Reply:
He was probably wanting other attributes from it but then decided to use `setReviewerId(-1)` instead.